AlfrescoApp/Model/Managers/AccountConfiguration.h

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (C) 2005-2016 Alfresco Software Limited.
```

Comment:
Only worth updating if you're adding anything else to this PR, but for new files this should be "2005-2017"